lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -450,9 +451,8 @@ void swift::ide::printCodeCompletionResultSourceText(
   }
 }
 
-void swift::ide::printCodeCompletionResultFilterName(
-    const CodeCompletionResult &Result, llvm::raw_ostream &OS) {
-  auto str = Result.getCompletionString();
+static void printCodeCompletionResultFilterName(
+    const CodeCompletionString *str, llvm::raw_ostream &OS) {
```

Comment:
Seems like we assume this is not null, `&` instead? There's other functions that take a `CodeCompletionString *` as well, not sure if it applies there too.

Reply:
I agree `&` makes more sense. I just wanted to keep the diff small.

Reply:
Fair enough, can always do as a follow up. I'm just worried about possible nullptr derefs since this does seem to be null in *some* places.